backend/dataall/modules/s3_datasets/cdk/dataset_stack.py

```diff
@@ -11,10 +11,12 @@
     Duration,
     CfnResource,
     CustomResource,
+    RemovalPolicy,
     Tags,
 )
 from aws_cdk.aws_glue import CfnCrawler
 
+from backend.dataall.base.utils.naming_convention import NamingConventionPattern, NamingConventionService
```

Comment:
@mourya-33 caught one last bug when testing need to import as 
```
from dataall.base.utils.naming_convention import NamingConventionPattern, NamingConventionService
```
 
removing `'backend.'` from import path

Reply:
Done Noah